TRANSLATING.md

```diff
@@ -0,0 +1,74 @@
+# 🌐 <abbr title="translation">Translation</abbr> Guide
```

Comment:
Why an abbreviation?

Reply:
I copy-pasted this from another repo that had a document for translating. Now that I look at it, there's no reason 😓 I will remove it.